godtools/Managers/GTDataManager.swift

```diff
@@ -13,8 +13,18 @@ import Spine
 import CoreData
 
 class GTDataManager: NSObject {
+    let documentsPath: String
+    let resourcesPath: String
     
     let serializer = Serializer()
+    let context = NSManagedObjectContext.mr_default()
+    
+    override init() {
+        documentsPath = NSSearchPathForDirectoriesInDomains(.documentDirectory, .userDomainMask, true)[0]
```

Comment:
So this one's a little interesting. It requires me to unwrap the value returned by `.first` or provide a default. Unwrapping the value is just as risky as using `[0]` and i don't know what default to provide. What do you think?